tests/dag_processing/test_dag_bundles.py

```diff
@@ -431,6 +429,7 @@ def test_view_url(self, mock_gitrepo, repo_url, expected_url, session):
             name="test",
             tracking_ref="main",
         )
+        bundle.initialize()
```

Comment:
We can't initialize the bundle in order to show the url - we definitely don't want to be pulling dag files down on the webserver, for example!